### ISSUE TITLE
Unable to complete helm install

### ISSUE THREAD

**Issue by hahobas** [devicehive/devicehive-docker#53]
```
[hahobas@localhost k8s]$ helm dependency update ./devicehive
Hang tight while we grab the latest from your chart repositories...
...Unable to get an update from the "local" chart repository (http://127.0.0.1:8879/charts):
        Get http://127.0.0.1:8879/charts/index.yaml: dial tcp 127.0.0.1:8879: connect: connection refused
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 3 charts
Downloading postgresql from repo https://kubernetes-charts.storage.googleapis.com/
Downloading kafka from repo https://kubernetes-charts-incubator.storage.googleapis.com/
Downloading redis from repo https://kubernetes-charts.storage.googleapis.com/
Deleting outdated charts
[hahobas@localhost k8s]$ helm install ./devicehive/
Error: apiVersion "rbac.authorization.k8s.io/v1" in devicehive/templates/devicehive-rolebinding.yaml is not available

```
Attempting to bypass by disabling rbac I get the following:
```
[hahobas@localhost k8s]$ helm install ./devicehive --set rbac.create=false
Error: [unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta1"]
```

I'm using helm version:
`Client: &version.Version{SemVer:"v2.9.0", GitCommit:"f6025bb9ee7daf9fee0026541c90a6f557a3e0bc", GitTreeState:"clean"}`
and kubernetes version:
`Kubernetes v1.9.1`
and using Fedora 27.

I've tried both on the master branch and the development branch.

Thank you for any help you can provide.

**Comment by sitnik** [comment]
Hi @hahobas, 
apiVersion "rbac.authorization.k8s.io/v1" went out of beta in Kubernetes 1.8 (https://v1-8.docs.kubernetes.io/docs/api-reference/v1.8/#rolebinding-v1-rbac). It definitely must be available in v1.9 clusters.
I've tested DeviceHive chart deployment in Kubernetes v1.9.7 cluster running in GKE and it deployed just fine without this issue.
Could you please provide full output of `kubectl version`, with 'Server Version:' string, and describe where and how your k8s cluster is deployed (GKE, AWS/kops, on premises, etc).

Thank you for report and testing.

**Comment by hahobas** [comment]
I'm using:
Server Version: version.Info{Major:"1", Minor:"6", GitVersion:"v1.6.0", GitCommit:"fff5156092b56e6bd60fff75aad4dc9de6b6ef37", GitTreeState:"dirty", BuildDate:"2017-04-07T20:43:50Z", GoVersion:"go1.7.1", Compiler:"gc", Platform:"linux/amd64"}

I can see that this is the origin of the problem so I consider this issue resolved.
@sitnik, Thank you for your help.
